.circleci/config.yml

```diff
@@ -44,10 +44,12 @@ jobs:
           command: *store_stripe_version
 
       - restore_cache:
+          name: Restore Fabric SDK
           keys:
             - fabric-sdk-cache-{{ checksum "fabric_version.txt" }}
 
       - restore_cache:
+          name: Restore Stripe SDK
```

Comment:
Note to self: Stripe caching should have been removed in #658. Will open a separate PR for that.

Reply:
#685